cvmfs/cvmfs_config

```diff
@@ -1117,6 +1118,28 @@ cvmfs_bugreport() {
    echo "JIRA bug report to https://sft.its.cern.ch/jira/browse/CVM"
 }
 
+cvmfs_fsck() {
+  
+  if ! cvmfs_readconfig; then
+    echo "Failed to read CernVM-FS configuration"
+    return 1
+  fi
+
+  local repo_list
+  repo_list=""
+  if [ "x$CVMFS_REPOSITORIES" != "x" ]; then
+    repo_list=`echo $CVMFS_REPOSITORIES | sed 's/,/ /g'`
+  fi
+
+  local cache_dir
+  cache_dir=$CVMFS_CACHE_BASE
+  local repo
+  for repo in $repo_list
+  do
+     sudo /usr/bin/cvmfs_fsck ${cache_dir}/${repo}
```

Comment:
Would be better with quotes, too (`sudo /usr/bin/cvmfs_fsck "${cache_dir}/${repo}"`)